ui/src/app/utils/user-apps-utils.spec.tsx

```diff
@@ -99,4 +104,38 @@ describe('User Apps Helper functions', () => {
     jest.advanceTimersByTime(20e3);
     expect(spyListAppsAPI).toHaveBeenCalledTimes(2);
   });
+
+  it('Opens Config panel if RStudio App is not running', async () => {
+    const navigate = mockNavigate;
+    expect(setSidebarActiveIconStore.value).toBeNull();
+
+    userAppsUtils.openRStudioOrConfigPanel(
+      'ws',
+      'wsid',
+      [{ status: AppStatus.STARTING, appType: AppType.RSTUDIO }],
+      navigate
+    );
+    expect(mockNavigate).not.toBeCalled();
+    // Since RStudio is NOT in running state this will open the RStudio config side panel and
```

Comment:
good tests!  I think this comment makes more sense 1 line above here

Reply:
Done